routes/conversation.js

```diff
@@ -60,6 +60,31 @@ router.get('/', auth, async (req, res) => {
   }
 });
 
+//POST create a new message
+
+router.post('/message', auth, async (req, res) => {
+  const msgData = req.body;
+  try {
+    const conversation = await Conversation.findById(msgData.room);
+
```

Comment:
Based off of conversation.members, you can figure out what languages you need to translate to

if members: ["Bonnie", "Kevin"]

` msgData.primaryLanguage` is "English"

so because Bonnie's language is french, we need to translate this to french 

`conversation.translation["french"] = translated_text`